aksel.nav.no/website/sanity/schema/objects/templates/ExampletextBlock.tsx

```diff
@@ -65,8 +65,6 @@ export const ExampletextBlock = defineType({
     },
   },
   components: {
-    preview: (values: Omit<ExampletextBlockT, "_key">) => (
-      <AkselExampletextBlock node={values} />
-    ),
+    preview: (values) => <AkselExampletextBlock node={values as any} />,
```

Comment:
Seems like they have messed up the returntypes in `preview` 😞

Reply:
😅
```suggestion
    preview: (values) => (
      <AkselExampletextBlock
        node={{ ...values, title: values.title?.toString() }}
      />
    ),
```